packages/core/stories/card.stories.tsx

```diff
@@ -66,17 +66,17 @@ export const All: ComponentStory<typeof Card> = () => (
 export const Default: ComponentStory<typeof Card> = () => (
   <Card>
     <div>
-      <h1>This is Card</h1>
-      <span>Using Nested DOM Elements</span>
+      <H1>This is Card</H1>
+      <Text>Using Nested DOM Elements</Text>
     </div>
   </Card>
 );
 
 export const Disabled: ComponentStory<typeof Card> = () => (
   <Card data-testid="card-disabled-example" disabled>
     <div>
```

Comment:
@alycrys @joshwooding Should we provide an alternative helper class that can be added to surrounding divs to avoid having to add disabled to each...?

Reply:
I think it's worth having a discussion about this. Maybe it'll be an enhancement and we keep it as is for now?....